.github/vale/styles/Aiven/capitalization_headings.yml

```diff
@@ -36,6 +36,7 @@ exceptions:
   - Elasticsearch
   - European Union
   - Flink
+  - GiB
```

Comment:
Could we double check the units intended here? GB would be gigabytes (which is what I expect, this is a standard measure), but GiB is for gibibytes ... is that correct?

Reply:
@packi , could you please confirm?

Reply:
We actually check for 4 GiB in the code